### PR TITLE
[Front] Add a button for copy to clipboard the current room id 

### LINKED ISSUE #209: [Front] add a clipboard button on the share room id popup
I want to add a copy to clipboard button for copy directly the room id and share it more easier .
You can aslo add this clipboard [icon](https://feathericons.com/?query=clipboard).
I have found a tuto for make a clipboard button see [here](https://www.w3schools.com/howto/howto_js_copy_clipboard.asp)

### PR DESCRIPTION
**Pull request by yannis-mlgrn**
Fixes #209 
**Features :** 
* added a button on the `share room id` popup
* added a beautiful css for the button

**Rendering :**
suppr

### CODE REVIEW

**Comment by yannis-mlgrn** [comment]
the gradient can be not in chords with the other css. I must  modifie the css for remove the gradient end put the same background color that the others buttons ?

File: front/src/components/ActivityBar.vue [hunk 1]
```diff
@@ -173,4 +183,56 @@ export default defineComponent({
...
+
+.copy_button:focus {
+    outline: none;
+}
+
+.copy_button.gradient {
+    background-image: linear-gradient(to right, #25aae1, #40e495, #30dd8a, #2bb673);
+    box-shadow: 0 4px 15px 0 rgba(49, 196, 190, 0.75);
+}
+.copy_wrapper[data-v-04a542f8] {
```
Comment (viandoxdev) [inline front/src/components/ActivityBar.vue]: Why `data-v-....` ?

**Review by viandoxdev — CHANGES_REQUESTED** [review]
LGTM, apart from the weird data thing

File: front/src/components/ActivityBar.vue (hunk 1 above)
Comment (yannis-mlgrn) [reply front/src/components/ActivityBar.vue]: idk why there is that lol

**Review by viandoxdev — APPROVED** [review]

**Pull request merged**